hat/assets/js/apps/Iaso/domains/periods/components/PeriodPicker.js

```diff
@@ -74,9 +74,16 @@ const PeriodPicker = ({
         setCurrentPeriod(newPeriod);
         onChange(getPeriodPickerString(periodType, newPeriod, value));
     };
-    if (!periodType) {
-        return null;
-    }
+
+    const handleChangeDay = useCallback(
+        date => {
+            const newValue = date?.format('YYYYMMDD') ?? null;
+            setCurrentPeriod(Period.parse(newValue)?.[1]);
```

Comment:
this is crashing if providing a invalid date: select a day and edit it manually, do a backspace => white page

https://user-images.githubusercontent.com/12494624/200288507-0d6b8881-383e-4535-9693-636fb763d445.mov